Xamarin.Forms.Platform.iOS/ModalWrapper.cs

```diff
@@ -112,5 +120,17 @@ public override UIViewController ChildViewControllerForStatusBarStyle()
 		{
 			return ChildViewControllers?.LastOrDefault();
 		}
+
+		void OnModalPagePropertyChanged(object sender, PropertyChangedEventArgs e)
+		{
+			if (e.PropertyName == Page.ModalBackgroundColorProperty.PropertyName)
+				UpdateBackgroundColor();
+		}
+
+		void UpdateBackgroundColor()
+		{
+			Color modalBkgndColor = ((Page)_modal.Element).ModalBackgroundColor;
+			View.BackgroundColor = modalBkgndColor.IsDefault ? UIColor.White : modalBkgndColor.ToUIColor();
```

Comment:
What's the advantage to having ModalBackgroundColor? Opposed to just doing something like this
```C#
Color modalBkgndColor = ((Page)_modal.Element).BackgroundColor;
View.BackgroundColor = modalBkgndColor.IsDefault ? UIColor.White : modalBkgndColor.ToUIColor();
```

Reply:
Actually I see why now... 
Is there ever a reason to set the ModalBackgroundColor and BackgroundColor? Or will you always set one of them to Transparent?

If we just added an Attached Property to indicate if the Modal should be transparent or not

```C#
bool Page.ModalBackgroundIsTransparent
```

Would that achieve the same thing?

Reply:
Could we just set it to Transparent if you use anything except FullScreen for the modal behavior?